contrib/scripts/install-app.sh

```diff
@@ -1,13 +1,10 @@
 #!/bin/bash
-
 set -e
-
 BUILD_NAME=catalog-harvest-registry-${REGISTRY_VERSION}
-
 echo "[-] Installing application"
 mkdir -p ${APP_DIR}
-wget -O ${APP_DIR}/${BUILD_NAME}.tar.gz https://s3.amazonaws.com/asa-dev/releases/${BUILD_NAME}.tar.gz
```

Comment:
I should have done `git mv` instead here.  We should consider symlinking the tarball instead of copying it to keep the resulting image size down.